BAZProjectC1/Extensions/UIImageViewExtension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIImageViewExtension.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 17/09/22.
+//
+
+import Foundation
+import UIKit
+extension UIImageView{
+    public func loadImageFromUrl(urlString: String){
+        if let url = URL(string: urlString) {
```

Comment:
Podemos cambiar este if por un guard let?
```suggestion
        guard let url = URL(string: urlString) else { return } 
```